### ISSUE TITLE
Send Notification: To Shut down the old Data Dictionary after we move Query Builder to new service

### ISSUE THREAD

**Issue by jpvajda** [newrelic/docs-website#1582]
### Issue

Once we complete the following issues: 

#1443
#1503 

We can shut down the old data dictionary service.  We'll want to give a few weeks of gap time between the changes we are making in the above issues and actually shutting down the old service. 

### Other Uses

The new data dictionary service covers all use cases. The legacy DDS just serves two cases:
Get all event types with their attributes nested inside

`{discovery_path:public-data-dictionary-service}/v1/definitions/events`

The same but for a single event

`{discovery_path:public-data-dictionary-service}/v1/definitions/events/{eventTypeName}`
### Acceptance Criteria

- [x] Validate the new integration points with the Query Builder and Event Explorer are working OK.
- [x] Contact  #help-dashboard-services and ask the PM / EM for any concerns to deprecate this service.
- [x] Draft an EOL nerdlife post, and link to the new data dictionary nerd life post
- [ ] Notify teams that may be using the old Dictionary Service, this will be shutdown and set an EOL date 2 weeks from the notice. 

POST IN: slack channels:

- #dev
- #wizards-programmabilty 
- #documentation

**Comment by aswanson-nr** [comment]
Added another issue to tackle removing the service after the 2 week notice #1770

**Comment by jpvajda** [comment]
I'm working on how we'll proceed with this shutdown with #help-dashboard-services

**Comment by jpvajda** [comment]
Internal Ticket: https://newrelic.atlassian.net/browse/DSERVICES-1181

**Comment by jpvajda** [comment]
Still confirming the official shutdown date, I propose May 20th From the Dashboard Team...

- We can support the deprecation from our side (action to shut-down the old service)
- If you expect our team to do the analysis about” if any other teams / services are relying on the old data dictionary service” prior to the depreciation date, we will need additional time than 10 May, as our current sprint is full
- In general, the timeline of “April 27 to May 10” is only a couple of weeks away, which gives no time for teams to plan in advance. They will all be forced to do this migration in their current or next sprint (not much flexibility). If you can, I would encourage you to give 2 sprints lead time for teams to plan, from the moment you send the comms out

**Comment by jpvajda** [comment]
Still waiting on confirmation from the Dashboards team how we'll proceed on this.

**Comment by jpvajda** [comment]
@jpvajda to follow up on this today...

**Comment by jpvajda** [comment]
This is unblocked now..

**Comment by jpvajda** [comment]
Nerdlife post is up, May 20th is the target for this work.
